src/app/beer_garden/garden.py

```diff
@@ -212,6 +212,45 @@ def update_garden(garden: Garden) -> Garden:
     return db.update(garden)
 
 
+def garden_sync(sync_target: str = None):
+    """Do a garden sync
+
+    If we're here it means the Operation.target_garden_name was *this* garden. So the
+    sync_target is either *this garden* or None.
+
+    If the former then call the method to publish the current garden.
+
+    If the latter then we need to send sync operations to *all* known downstream
+    gardens.
+
+    Args:
+        sync_target:
+
+    Returns:
+
+    """
+    # If a Garden Name is provided, determine where to route the request
+    if sync_target:
+        logger.debug("Processing garden sync, about to publish")
+
+        publish_garden()
+
+    else:
+        from beer_garden.router import route
+
+        # Iterate over all gardens and forward the sync requests
+        for garden in get_gardens(include_local=False):
+            logger.debug(f"About to create sync operation for garden {garden.name}")
+
+            route(
```

Comment:
If you add `import beer_garden` to the top, you can run this command as `beer_garden.router.router(`. I know you did this because of the import error because they reference each other. This is just my new trick to getting around it.

Reply:
Hmm, I'm not sure if that's better or worse?